dojo/api.py

```diff
@@ -89,7 +95,85 @@ def form_args(self, bundle):
         return kwargs
 
 
+def sanitize(text):
+    # We put the single quotes back, due to their frequent usage in exception
+    # messages.
+    return escape(text).replace('&#39;', "'").replace('&quot;', '"').replace('&#x27;', "'")
+
+
 class BaseModelResource(ModelResource):
+
+    def wrap_view(self, view):
+        """
+        Wraps views to check if APIv1 is enabled
+        """
+        @csrf_exempt
+        def wrapper(request, *args, **kwargs):
+            try:
+                api_v1_deprecation_warning = 'APIv1 is deprecated and may contain vulnerabilities. It\'s disabled by default. '
+                if not settings.LEGACY_API_V1_ENABLE:
+                    raise BadRequest({'code': 666, 'message': api_v1_deprecation_warning + 'At your own risk it can be enabled by setting DD_LEGACY_API_V1_ENABLE to True, or by setting LEGACY_API_V1_ENABLE to True in  settings(.dist).py or local_settings.py'})
```

Comment:
:-D

Reply:
I would reword as `It can be enabled at your own risk by setting...`, or put a `,` after risk as it stands. Now, I didn't intend to put so many spelling comment in the review of that PR! ;-)